src/kOS/Module/kOSProcessor.cs

```diff
@@ -343,8 +345,8 @@ public void InitObjects()
             shared.Window.AttachTo(shared);
 
             // initialize archive
-            var archive = shared.Factory.CreateArchive();
-            shared.VolumeMgr.Add(archive);
+            Archive = shared.Factory.CreateArchive();
```

Comment:
Storing the `Archive` volume allows us to bypass the range check in "getting" the archive when switching to it.  While slightly hacky, I didn't want to make a major change to the system with a pending sub-directories PR out there, and the alternative was to add a new method for returning the volume without a range check, or checking the range without the volume object itself.